docs/loader.md

```diff
@@ -0,0 +1,173 @@
+the following modes: containerd (`container`), Firecracker (`firecracker`), and Firecracker with
+snapshots (`firecracker_snapshots`).Loader will be cloned on every node specified as argument of the cluster create
+script. The same holds for Kubernetes API server certificate.
+
+* To create a multi-node cluster, specify the node addresses as the arguments and run the following command:
+
+```bash
+$ bash ./scripts/setup/create_multinode.sh <master_node@IP> <loader_node@IP> <worker_node@IP> ...
+```
+
+This command will create the following setup: control plane is placed on master node, loader node is used for running
+loader and monitoring pods (mostly, Prometheus, if enabled in setup config), workers are used purely for working pods.
+In
+this setup, neither control plane nor workers are affected by loader and monitoring, creating more reliable measurements
+of performance.
+
+* Single-node cluster (experimental)
+
+```bash
+$ bash ./scripts/setup/create_singlenode_container.sh <node@IP>
+```
+
+This mode is only for debugging purposes, and there is no guarantees of isolation between the loader and the master-node
+components.
+
+### OpenWhisk cluster
+
+See the instructions located in `openwhisk_setup/README.md`.
+
+### Check cluster health (on the master node)
+
+Once the setup scripts are finished, we need to check if they have completed their jobs fully, because, e.g., there
+might be race conditions where a few nodes are unavailable.
+
+* First, log out the status of the control plane components on the master node and monitoring deployments by running the
+  following command:
+
+```bash
+$ bash ./scripts/util/log_kn_status.sh
+```
+
+* If you see everything is `Running`, check if the cluster capacity is stretched to the desired capacity by running the
+  following script:
+
+```bash
+$ bash ./scripts/util/check_node_capacity.sh
+```
+
+If you want to check
+the [pod CIDR range](https://www.ibm.com/docs/en/cloud-private/3.1.2?topic=networking-kubernetes-network-model), run the
+following
+
+```bash
+$ bash ./scripts/util/get_pod_cidr.sh
+```
+
+* Next, try deploying a function (`myfunc`) with the desired `scale` (i.e., the number of instances that must be active
+  at all times).
+
+```bash
+$ bash ./scripts/util/set_function_scale.sh <scale>
+```
+
+* One should verify that the system was able to start the requested number of function instances, by using the following
+  command.
+
+```bash
+$ kubectl -n default get podautoscalers
+```
+
+## Tune the timing for the benchmark function
+
+Before start any experiments, the timing of the benchmark function should be tuned so that it consumes the required
+service time more precisely.
+
+First, run the following command to deploy the timing benchmark that yields the number of execution units* the function
+needs to run given a required service time.
+
+* The execution unit is approximately 100 `SQRTSD` x86 instructions.
```

Comment:
can we get this replaced by a simple albeit cross-platform counter increment? please raise an issue